syntax = "proto3";

package helloworld;

service TestServer {
  rpc hello_request (HelloMessage) returns (Reply) {}
}

message Reply {
  int32 result = 1;
}

message HelloMessage {
  int32 a = 1;
  int32 b = 2;
}
